_includes/code/graphql.additional.py

```diff
@@ -1,37 +1,41 @@
-import weaviate
-import json
+# TEST DISABLED - sandbox needed
 
-client = weaviate.Client(
-    'https://edu-demo.weaviate.network',
-    auth_client_secret=weaviate.AuthApiKey('learn-weaviate'),
-)
+# import weaviate
+# import json
+
+# client = weaviate.Client(
+#     'https://edu-demo.weaviate.network',
+#     auth_client_secret=weaviate.AuthApiKey('learn-weaviate'),
+# )
 
 
-def test_gqlresponse(response_in, gqlresponse_in):
-    for i, result in enumerate(response_in['data']['Get']['JeopardyQuestion']):
-        assert result['question'] == gqlresponse_in['data']['Get']['JeopardyQuestion'][i]['question']
+# def test_gqlresponse(response_in, gqlresponse_in):
+#     for i, result in enumerate(response_in['data']['Get']['JeopardyQuestion']):
+#         assert result['question'] == gqlresponse_in['data']['Get']['JeopardyQuestion'][i]['question']
 
 
 # ===================
 # ===== Sorting =====
 # ===================
 # START Sorting Python
-response = (
-    client.query
-    .get('JeopardyQuestion', ['question', 'answer'])
+from weaviate.collections.classes.grpc import Sort
```

Comment:
We can remove this line from everywhere, since the line is imported up above with: # START-ANY

Reply:
It will not be hidden. The # START-ANY section is shown on every snippet.